packages/manager-react-components/src/components/templates/onboarding/onboarding.stories.tsx

```diff
@@ -5,6 +5,7 @@ import {
   OnboardingLayoutProps,
 } from './onboarding.component';
 import { Card } from '../../navigation/card/card.component';
+import placeholderSrc from './assets/placeholder.png';
```

Comment:
I known it's done in app with this nomenclature.

But why don't use constante nomenclature for external image imported ? 
```suggestion
import PLACE_HOLDER_SRC from './assets/placeholder.png';
```

Reply:
Why use "constante nomenclature" actually ?
We rarely mutate variable in React apps

Reply:
To known it's external value declared on top of file. Not a params or a local variable (As declared as const)

It's for differentiate scope directly when we read the code.